tier/helm/tier/deps.yaml

```diff
@@ -0,0 +1,28 @@
+apiVersion: plural.sh/v1alpha1
+kind: Dependencies
+metadata:
+  application: true
+  description: Deploys tier crafted for the target cloud
+spec:
+  dependencies:
+  - type: helm
+    name: bootstrap
+    repo: bootstrap
+    version: '>= 0.5.1'
+  
```

Comment:
could you rm these empty lines

Reply:
FYI: I didn't change this file, this is what's generated from `plural create`